android/src/main/java/nl/xguard/flic2/Flic2.java

```diff
@@ -171,28 +172,32 @@ public void stopScan() {
 
     @Override
     public void onHostResume() {
+        Context context = getReactApplicationContext();
+        boolean isRunning = isServiceRunning(context, Flic2Service.class);
+        if (isRunning) {
+            Intent intent = new Intent(context, Flic2Service.class);
+            getReactApplicationContext().bindService(intent, mFlic2ServiceConnection, BIND_AUTO_CREATE);
+        }
     }
 
     @Override
     public void onHostPause() {
-        // empty
+        try {
+            Context context = getReactApplicationContext();
+            boolean isRunning = isServiceRunning(context, Flic2Service.class);
+            if (mFlic2ServiceConnection.isServiceConnected() && isRunning) {
+                context.unbindService(mFlic2ServiceConnection);
```

Comment:
wat is het effect van een unbind op onHostPause? Werkt de knop dan nog wel?

Reply:
Met de unbind wordt de directe verbinding tussen contact (lees activity) en service verbroken. 
De service is verantwoordelijk voor het afhandelen van de button events, deze worden na de initialisatie door ReactFlic2ButtonListener afgehandeld.

Na het afsluiten van de activity is de service nog actief, waardoor ReactFlic2ButtonListener events via Flic2ReactEvent kan doorsturen in het react framework

Reply:
👍 we builden nu een .apk voor test op de kapotte device